QuizApp/App/Common/Coordinator.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+final class Coordinator: CoordinatorProtocol {
+
+    var navigationController: UINavigationController
+
+    init(navigationController: UINavigationController) {
+        self.navigationController = navigationController
+    }
+
+    func showLogin() {
+        startLoginFlow()
+    }
+
+}
+
+// MARK: - Private methods
+private extension Coordinator {
+
+    func startLoginFlow() {
+        let loginViewController = LoginViewController()
+        navigationController.pushViewController(loginViewController, animated: true)
```

Comment:
What if some screen is in the app and you want to navigate to the Login page? Is push the correct way to do it?

Reply:
Agreed. I've changed it to use the `setViewControllers(_:animated:)` method. That way, in the example you mentioned, we would reset the navigation stack, without the need to pop view controllers.